content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/customheadlibs.amp.html

```diff
@@ -1 +1,16 @@
-<script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.2.js"></script>
```

Comment:
@marcos-chavez-bounteous, actually, we will need to completely remove this file. all amp related files will now live in /apps/core-components-examples. You can still merge like that and we will slowly migrate these files to their new location when needed, on a case by case basis.